infra/base-images/base-builder/install_python.sh

```diff
@@ -17,5 +17,7 @@
 
 echo "ATHERIS INSTALL"
 unset CFLAGS CXXFLAGS
-pip3 install -v --no-cache-dir "atheris>=2.0.6" "pyinstaller==4.1" "coverage==6.3.2"
+# PYI_STATIC_ZLIB=1 is needed for installing pyinstaller 5.0
+export PYI_STATIC_ZLIB=1
+pip3 install -v --no-cache-dir "atheris>=2.0.6" "pyinstaller==5.0.1" "coverage==6.3.2"
```

Comment:
btw I would have put this on line 22 instead of exporting it to reduce the chances of side effects.

Reply:
At this point maybe don't change it though since we will have to test everything again.